src/Geometry/Transform.jl

```diff
@@ -163,6 +147,7 @@ Transform(rotation::AbstractArray{S,2}, translation::AbstractArray{S,1})
 Transform{T} = SMatrix{4,4,T,16}
 export Transform
 
+lastrow(::Type{T}) where{T<:Real} = SMatrix{1,4,T}(zero(T),zero(T),zero(T),one(T)) 
```

Comment:
how about `lastrow(::Type{T} = Float64)` to conform with the rest of the code in Transform.jl?

i'll probably revisit these convenience functions later as i think they could be a little more systematic, but this is fine for now

Reply:
lastrow was only being used in one place so I removed the function and inlined it.